x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/jwt/JwtRealmSettings.java

```diff
@@ -131,8 +138,8 @@ private JwtRealmSettings() {}
      * Get all secure settings.
      * @return All secure settings.
      */
-    public static List<Setting.AffixSetting<SecureString>> getSecureSettings() {
-        return List.of(HMAC_JWKSET, HMAC_KEY, CLIENT_AUTHENTICATION_SHARED_SECRET);
+    public static Set<Setting.AffixSetting<SecureString>> getSecureSettings() {
```

Comment:
I think I made getNonSecureSettings() and getSecureSettings() public because they were called directly by test code in a previous PR. The test code is gone. I can change them to private.

Reply:
Changed to package private. Resolved.